### PR TITLE
Take a stab at documenting the SIG Roadmap

### PR DESCRIPTION
**Pull request by fdegir**
This commit creates the initial version of the SIG Roadmap based on the recent conversations that took place during the SIG Software Supply Chain Meetings.

The roadmap is not exhaustive and should be iterated further with the contributions from the community.

This roadmap is heavy influenced by the SIG Interoperability roadmap.

### CODE REVIEW

File: docs/roadmap.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+## Overview
+
+This document outlines the roadmap for the Continuous Delivery Foundation (CDF)
+Software Supply Chain Special Interest Group (SIG). The roadmap represents the
+initiatives of the open source collaborations. The roadmap does not specify
+timelines or delivery dates, but rather uses time horizons (now, next, later).
+
+## Charter
+
+The Software Supply Chain SIG aims to study different aspects of CI/CD systems, securing
```
Comment (bendory) [inline docs/roadmap.md:20]: Suggest: "The Software Supply Chain SIG aims to study different aspects of CI/CD systems and share best practices for CICD with a focus on securing..."

File: docs/roadmap.md [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+CDF [CDEvents](https://cdevents.dev) project works on creating a common specification for
+Continuous Delivery events in order to achieve interoperability within the CD ecosystem.
+The current version of the specification does not have Software Supply Chain related
+events and the SIG Software Supply Chain could look for ways to contribute such events
+to the project.
+
+Please note that this work depends on the earlier work, Establishing Shared Vocabulary.
+
+
+## Contributors
```
Comment (bendory) [inline docs/roadmap.md:118]: Is this intended to be "Contributors to this Document" or "Contributors to the SIG"? I assume the former, but it isn't clear -- suggest framing it that way for clarity.

**Review by bendory — CHANGES_REQUESTED** [review]
/LGTM modulo two suggestions

File: docs/roadmap.md [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+
+### Next
+
+#### Establishing Shared Vocabulary
+
+Various groups within the CDF such as [SIG Interoperability](https://github.com/cdfoundation/sig-interoperability)
+and [SIG Events](https://github.com/cdfoundation/sig-events) have been working on
+developing a shared vocabulary to help enable the further developments in corresponding
+areas. The work is done within the SIG Interoperability repository and the community is
+encouraged to contriubute to corresponding documents listed below.
```
Comment (developer-guy) [inline docs/roadmap.md:88]: *typo -> contribute 😁

**Review by developer-guy — APPROVED** [review]
LGTM

File: docs/roadmap.md (hunk 3 above)
Comment (fdegir) [reply docs/roadmap.md]: Thanks for catching it Batuhan! Updated.

**Comment by fdegir** [comment]
> I assume the `.swp` was included by accident and should be deleted from this PR?

Thanks! Removed now.

**Review by bendory — APPROVED** [review]

**Pull request merged**